crates/ra_hir/src/ty/infer.rs

```diff
@@ -1360,6 +1375,26 @@ impl<'a, D: HirDatabase> InferenceContext<'a, D> {
             _ => None,
         }
     }
+
+    fn resolve_ops_try_ok(&self) -> Option<TypeAlias> {
+        use crate::name::{OK, OPS, TRY};
```

Comment:
I think we should include these names locally to show intent (they are names), also to avoid pollute module with kind of arbitrary constants. @matklad

Reply:
I'd do `use crate::name`  at the module levenl and  then use `PathSegment { name: name::STD, ...}`

Reply:
We probably should also extract these "known paths" into a module of their own:

```
mod known_paths {
    pub(crate) fn try_trait() -> Path { .. }
}
```

Reply:
Let's make `known_paths` to different PR.